plugin/trino-iceberg/src/main/antlr4/io/trino/plugin/iceberg/sort/SortDefinition.g4

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+grammar SortDefinition;
+
+tokens {
+    DELIMITER
+}
+
+sortDefinition
+    : identifier direction=(ASCENDING | DESCENDING)? (NULLS nullOrder=(FIRST | LAST))? EOF;
```

Comment:
This can be easily parsed with a simple regex.
I've added a fixup commit proposing such a change, please take a look